Task2.py

```diff
@@ -0,0 +1,26 @@
+""" 2. List practice
+Используйте генератор списков чтобы получить
+следующий: ['ab', 'ac', 'ad', 'bb', 'bc', 'bd'].
+Используйте на предыдущий список slice чтобы получить
+следующий: ['ab', 'ad', 'bc'].
+Используйте генератор списков чтобы получить следующий
+['1a', '2a', '3a', '4a'].
+Одной строкой удалите элемент  '2a' из прошлого с
+писка и напечатайте его.
+Скопируйте список и добавьте в него элемент '2a'
+так чтобы в исходном списке этого элемента не было.
+"""
+import copy
+
+lst1 = [a + b for a in "ab" for b in ['b', 'c', 'd']]
```

Comment:
a, b - bad namings

Reply:
I agree. Now I try to give names to variables more meaningfully.